src/simulation/m_hypoelastic.fpp

```diff
@@ -175,7 +213,7 @@ contains
                         G_K_field(k, l, q) = G_K
 
                         !TODO: take this out if not needed
```

Comment:
remove commented line depending on what the actual state of affairs is

Reply:
Should stay